tasks/post-to-cmr/tests/data/payload.json

```diff
@@ -50,59 +47,21 @@
             "fileStagingDir": "file-staging",
             "url_path": "",
             "name": "BROWSE.MOD11A1.A2017200.h19v04.006.2017201090724.hdf"
+          },
+          {
+            "checksumType": "CKSUM",
+            "bucket": "cumulus-public",
+            "checksumValue": 4026661056,
+            "filename": "s3://cumulus-public/MOD11A1.A2017200.h19v04.006.2017201090724.cmr.xml",
```

Comment:
The .cmr.xml file is in both granules and inputFiles.  Should post-to-cmr take only list of granules?  The files from processing task should have been added to granules by the move-granules task.

Reply:
MoveGranules has to output the updated granules (Because it needs the same output as PostToCmr) so I thought Might as well use it.